ipld/retriever_test.go

```diff
@@ -6,15 +6,26 @@ import (
 	"testing"
 	"time"
 
+	"github.com/gammazero/workerpool"
+	format "github.com/ipfs/go-ipld-format"
 	mdutils "github.com/ipfs/go-merkledag/test"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/tendermint/tendermint/pkg/da"
+	"github.com/tendermint/tendermint/pkg/wrapper"
+
+	"github.com/celestiaorg/nmt"
+	"github.com/celestiaorg/rsmt2d"
 )
 
-func TestRetriever_Retrieve(t *testing.T) {
-	rand.Seed(time.Now().UnixNano()) // otherwise, the quadrant sampling is deterministic in tests
+func init() {
+	// randomize quadrant fetching, otherwise quadrant sampling is deterministic
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Maybe we should add a comment that we are aware that this is not (cryptographically) secure and that it's OK in this case that others can still guess which square we will pick here?

Reply:
It's just a test. So nvm. Resolving.